src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -163,13 +163,50 @@ describe('sidebar.components.sidebar-content', function() {
           auth: { status: 'unknown' },
         }
       );
-    })
-  );
+    });
+  };
+
+  beforeEach(() => {
+    makeSidebarContentController();
+  });
 
   afterEach(function() {
     return sandbox.restore();
   });
 
+  describe('showSelectedTabs', () => {
+    beforeEach(() => {
+      setFrames([{ uri: 'http://www.example.com' }]);
+      ctrl.search = { query: sinon.stub().returns(undefined) };
+    });
+
+    it('sets showSelectedTabs to false if there is a search query', () => {
```

Comment:
`showSelectedTabs` is a method that returns a value rather than one which sets/does something as a side effect, so I would suggest writing these descriptions as "returns {true, false} if {condition}"